persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java

```diff
@@ -148,14 +148,14 @@ public void start() {
                tableManager.updateMetaTable(connection);
             } else {
                // The meta table does not exist, therefore we must be reading from a 11.x store. Migrate the old data
-               org.infinispan.util.logging.Log.PERSISTENCE.startMigratingPersistenceData();
+               org.infinispan.util.logging.Log.PERSISTENCE.startMigratingPersistenceData(cacheName);
```

Comment:
offtopic:
can you remove the following fields? they are only used in the `start()` method and can be obtained from `ctx` and it saves some memory:
`globalConfiguration`
`cacheName`
`isDistributedCache`

Saving (computed by JOL plugin):
compressed oops: 64 bytes => 56 bytes
8bytes align: 120 bytes => 96  bytes

Reply:
Done